src/swap/SwapScreen.tsx

```diff
@@ -62,7 +63,126 @@ const DEFAULT_SWAP_AMOUNT: SwapAmount = {
   [Field.TO]: '',
 }
 
-type Props = NativeStackScreenProps<StackParamList, Screens.SwapScreenWithBack>
+interface SwapState {
+  fromTokenId: string | undefined
+  toTokenId: string | undefined
+  // Raw input values (can contain region specific decimal separators)
+  swapAmount: SwapAmount
+  updatedField: Field
```

Comment:
what about `lastUpdatedField`?

Reply:
Good idea!

Reply:
Backtracking a bit on this, I started making the change but then noticed some [other structures use `updatedField`](https://github.com/valora-inc/wallet/blob/c55a5c78335565f75193a06fb42f567667cf194a/src/swap/types.ts#L23), where `lastUpdatedField` wouldn't really apply I think.
So leaving as is, to keep consistency.